go.sum

```diff
@@ -1,5 +1,10 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
looks like a little `go mod tidy` wouldn't go amiss

Reply:
Thought the same, but no, these are all still there afterwards.